test/Sema/struct_differentiable.swift

```diff
@@ -414,6 +414,22 @@ extension NoMemberwiseInitializerExtended: Differentiable
 extension NoMemberwiseInitializerExtended: EuclideanDifferentiable
   where T : Differentiable & AdditiveArithmetic {}
 
+// TF-1190: Test `@noDerivative` warning for property wrapper backing storage properties.
+
+@propertyWrapper
+struct Wrapper<Value> {
+  private var value: Value
+  var wrappedValue: Value {
+    value
+  }
+}
+struct TF_1190<T> {}
+struct TF_1190_Outer: Differentiable {
+  // expected-warning @+1 {{stored property '_x' has no derivative because 'Wrapper<TF_1190<Float>>' does not conform to 'Differentiable'; add an explicit '@noDerivative' attribute}}
```

Comment:
Note: perhaps we could improve this error message for property wrapper backing storage properties. I didn't take time to think of a good message, since the fix-it is very clear and actionable.

```console
tf-1190.swift:10:3: warning: stored property '_x' has no derivative because 'Wrapper<Tensor<Float>>' does not conform to 'Differentiable'; add an explicit '@noDerivative' attribute
  @Wrapper var x: Tensor<Float>
  ^
  @noDerivative
```